frontend/src/pages/Resources/components/ResourceSharingForm/ResourceSharingForm.jsx

```diff
@@ -1,132 +1,273 @@
-import React from "react";
+import React, { useState } from "react";
+import Joi from "joi-browser";
 import { Button2 } from "../../../../components/util/Button/index";
 import style from "./resource-sharing-form.module.scss";
 
 export function ResourceSharingForm() {
+  const [formdata, setFormData] = useState({
+    name: "",
+    email: "",
+    link: "",
+    description: "",
+    trust: null,
+    dob: "",
```

Comment:
instead of `dob` please change it to `Date of Validation` or `dov`

Reply:
I have implemented your proposed changes. Please have a look
![image](https://user-images.githubusercontent.com/70805691/111144201-c6233f80-85ac-11eb-859b-c318c95f6c01.png)